turbo-repo/apps/web/app/globals.css

```
@@ -0,0 +1,106 @@
+     etc.), con más especificidad que un selector de elemento plano; !important
+     es necesario para ganarles sin tener que igualar esa especificidad.
+     Quitamos el ring en focus y mostramos un borde solo mientras el botón
+     está presionado. */
+  button:focus {
+    outline: none !important;
+    box-shadow: none !important;
+  }
+
+  button:active {
+    border-width: 1px !important;
+    border-color: currentColor !important;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)